nvflare/fuel/f3/cellnet/cell.py

```diff
@@ -1477,20 +1438,19 @@ def _process_request(self, origin: str, message: Message) -> Union[None, Message
             self.logger.debug(f"{self.my_info.fqcn}: invoking incoming request filters")
             assert isinstance(req_filters, list)
             for f in req_filters:
-                assert isinstance(f, _CB)
+                assert isinstance(f, Callback)
```

Comment:
If we need this check on production code, we need to change assert to ```if not isinstance(_cb, Callback): raise ValueError```

If we don't need, we should remove these assert statements.

Reply:
Assert can be useful for developers. It's used everywhere. Let's leave it here for now.